mautic/mautic
# When merging contacts, all UTM Tags are saved

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
When merging contacts, UTM tags from a looser contact are not saved. This PR fixes it.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a form with the email field.
3. Add the form to a landing page.
4. Visit a landingpage with a UTM parameter.
5. A new anonymous contact is created. The UTM Tags are recorded.
6. Fill a form on the landingpage with the email of existing contact with UTM tags. 
7. All UTM tags are saved. Before UTM tags of the merged contact was removed.


Visit a landingpage with a UTM parameter:

A new anonymous contact is created. The UTM Tags are recorded

The contact fills a form on the landingpage and by that she is identified as an existing contact

The UTM Tag from the anonymous user is not visible on the existing contact.
